### PR TITLE
Add design customization

### PR DESCRIPTION
**Pull request by oleksandrbulakh**

### CODE REVIEW

File: src/components/mixins/typography.js [hunk 1]
```diff
@@ -1,85 +1,109 @@
...
+      props.theme.colors.coverBgOverlayTextColor
+    )};
   ${media.tablet`
-    font-size: 46px;
+    ${props => mediaFontSize(props.theme.fonts.Heading1.size, 0.92)};
   `};
   ${media.mobile`
-    font-size: 30px;
+    ${props => mediaFontSize(props.theme.fonts.Heading1.size, 0.6)};
     line-height: 34px;
```
Comment (YuliaSavchuk) [inline src/components/mixins/typography.js]: Should line-height be fixed? or 0.6 also?

File: src/components/mixins/typography.js [hunk 2]
```diff
@@ -1,85 +1,109 @@
...
   `};
 `;
 
 export const mainFontH2 = css`
-  font-size: 40px;
+  ${props => styledFonts(props.theme.fonts.Heading2)};
```
Comment (YuliaSavchuk) [inline src/components/mixins/typography.js]: props.theme is an object. why does it have Heading in upper case?

File: src/components/questions/Statement/Statement.styled.js [hunk 3]
```diff
@@ -10,7 +10,17 @@ export const AnswerText = styled(Text)`
   font-weight: bold;
 `;
 
-export const StatementCaptionContainer = styled.div`
+export const StatementCaptionContainer = styled.li`
```
Comment (YuliaSavchuk) [inline src/components/questions/Statement/Statement.styled.js]: li? it looks like a wrapper, not a list item

File: src/components/questions/Statement/Statement.js [hunk 4]
```diff
@@ -81,7 +82,7 @@ export class StatementChoice extends Component {
                   </TextWrapper>
                 </StatementCaption>
               </StatementCaptionWrapper>
-            </StatementCaptionContainer>
+            </StatementCaptionMobileContainer>
```
Comment (YuliaSavchuk) [inline src/components/questions/Statement/Statement.js]: Too many *Statement* in class names. It makes this code hard to read. It's clear enough for me that this is statement question :)

File: public/manifest.json [hunk 5]
```diff
@@ -64,17 +60,22 @@
           ],
           "background": {
             "header": {
-              "brightness": 0,
+              "brightness": null,
```
Comment (YuliaSavchuk) [inline public/manifest.json]: why null values are used? it's a bit confusing

File: src/components/questions/Statement/Statement.styled.js [hunk 6]
```diff
@@ -42,8 +52,8 @@ export const StatementCaption = styled.div`
   text-align: center;
   margin-right: 7px;
   margin-bottom: 18px;
-  font-family: Averta, Arial, Sans-Serif;
-  font-size: 13px;
+  font-family: Montserrat, Arial, Sans-Serif;
```
Comment (oleksandrbulakh) [inline src/components/questions/Statement/Statement.styled.js]: use secondary font mixin

File: src/components/questions/Question.styled.js [hunk 7]
```diff
@@ -12,5 +12,8 @@ export const QuestionContainerOverlay = styled.div`
   bottom: 0;
   left: 0;
   display: ${props => (props.isShown ? 'block' : 'none')};
-  background-color: ${props => transparentize(props.theme.colors.mainBgColor, 0.5)};
+  background-color: ${props =>
```
Comment (YuliaSavchuk) [inline src/components/questions/Question.styled.js]: transparentize(props.bodyBackground && props.bodyBackground.color ? props.bodyBackground.color : '#fff', 0.5)

File: public/manifest.json [hunk 8]
```diff
@@ -29,10 +29,6 @@
               "key": "cta-button-color",
               "value": "#271100"
             },
```
Comment (YuliaSavchuk) [inline public/manifest.json]: Саня, я не в темі, але куди дівся main-bg-color? вже не треба?

**Pull request merged**